applications/mne_anonymize/mne_anonymize.pro

```diff
@@ -40,15 +40,41 @@ include(../../mne-cpp.pri)
 
 TEMPLATE = app
 
-QT += widgets concurrent network
+QT += core gui widgets
 
 CONFIG   += console
 
+CONFIG(debug,debug|release) {
+    macx {
+        CONFIG -= app_bundle
+    }
+}
+
+contains(MNECPP_CONFIG, noOpenGL) {
+    DEFINES += NO_OPENGL
+    QT -= opengl
+}
+
 contains(MNECPP_CONFIG, static) {
     CONFIG += static
     DEFINES += STATICBUILD
 }
 
+contains(MNECPP_CONFIG, wasm) {
+#    QMAKE_LFLAGS += -s ERROR_ON_UNDEFINED_SYMBOLS=1
+#    QMAKE_LFLAGS += -s ASSERTIONS=1
+#    QMAKE_LFLAGS += -s STRICT=0
+#    QMAKE_LFLAGS += -s FORCE_FILESYSTEM=1
```

Comment:
try setting `QMAKE_LFLAGS += -s TOTAL_MEMORY=4GB` here. Afterwards you can try pushing your changes to your remote via `git push origin mne_anonymize_GUI:wasm`. The command will create a new wasm branch on your remote, if not already existing. This will trigger the CI to build the wasm version and host it on your gh-pages. Once the CI finishes you should be able to access your application via [https://svdecomposer.github.io/mne-cpp/mne_anonymize.html](https://svdecomposer.github.io/mne-cpp/mne_anonymize.html). This way we can check which web browser actually support the 4GB WASM memory flag.

Reply:
Hi, 
 - I'm trying to understand what do you mean by "here". I've moved the arg definitions upwards with the rest of them. Not sure that's what you meant. I've commented because I've tried a few things and it wasn't quite working. I didn't want to spend that much time on this at that point. 

 - Managed to get the wasm working locally. Both qt and mne-cpp compiling just fine in darwin!!! ✌🏻
 - Modified a bit the inner workings. Now you should be able to anonymize two files of ± 1.99GB or so (4gb - the footprint of the application which shouldn't be that much). The only con is that every time you anonymize a file you need to re-load because immediately after anonymization finishes, the **internal** copy of the input file gets deleted.

Reply:
ah just saw you tried the flag below already. sry for the noise. were you able to see the flag int the cmd line output when compiling with emscripten? did it error during compilation?